deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr_ocp.yaml

```diff
@@ -1,67 +0,0 @@
-apiVersion: csi.ibm.com/v1
```

Comment:
why removing the ocp cr?

Reply:
it is a temporary yaml [used for non-x86 ocp](https://www.ibm.com/support/knowledgecenter/SSRQ8T_1.3.0/csi_ug_install_operator_github.html) when the community sidecars were not multi-arch